doc/UserGuide.md

```diff
@@ -486,6 +486,40 @@ examples.
 There is also `Gen.nonEmptyContainerOf` for generating non-empty containers, and
 `Gen.containerOfN` for generating containers of a given size.
 
+To generate a container by picking an arbitrary number of elements use
+`Gen.someOf`, or by picking one or more elements with
+`Gen.atLeastOne`.
+
+```scala
+val zeroOrMoreDigits = Gen.someOf(1 to 9)
+
+val oneOrMoreDigits = Gen.atLeastOne(1 to 9)
+```
+
+Here are generators that randomly pick `n` elements from a container
+with `Gen.pick`:
+
+```scala
+val fiveDice: Gen[Seq[Int]] = Gen.pick(5, 1 to 6)
+
+val threeLetters: Gen[Seq[Char]] = Gen.pick(3, 'A' to 'Z')
+```
+
+Note that `Gen.someOf`, `Gen.atLeastOne`, and `Gen.pick` only randomly
+select elements, they do not generate permutations of the selected
+elements in different orders.  Selection is random, but the order is
```

Comment:
Observed behavior of `pick` is that `pick(2, 1 to 10)` generates (for large samples) 73 distinct 2-lists, out of a total of 90 possible. 73 < 90 is consistent with the remark that `pick` does not, in general, permute all its results. 

But since there are only 45 (=10*9/2) distinct 2-sets from (1 to 10), 73 > 45 implies that `pick` __does permute__ some of its results, so the order is __not deterministic__. For instance `List(4, 5)` and `List(5, 4)` are both generated by `pick(2, 1 to 10)`.

Reply:
True, using the word "deterministic" is confusing.   I'll remove it.